dask/array/chunk.py

```diff
@@ -9,6 +9,7 @@
 
 from dask.core import flatten
 
+identity = 9223372036854775807
```

Comment:
You're using this value (2^63-1) because np.array requires an int for initial, right? You'll want to use sys.maxsize / -sys.maxsize instead. Maybe just put it inline rather than assigning to a variable here.